src/test/scala/org/clulab/wm/eidos/text/eval6/TestDoc2.scala

```diff
@@ -1,27 +1,190 @@
+      val foodInsecurity = NodeSpec("Food insecurity")
+      val conflict = NodeSpec("conflict")
+      val fams = NodeSpec("families", Quant("many"))
+
+      tester.test(foodInsecurity) should be (successful)
+      tester.test(conflict) should be (successful)
+      tester.test(fams) should be (successful)
+
+      // TODO: we really need to decide how to handle nested events!! (we should have "families to leave")
+      tester.test(EdgeSpec(foodInsecurity, Causal, fams)) should be (successful)
+      tester.test(EdgeSpec(conflict, Causal, fams)) should be (successful)
+    }
+  }
+
+  { // Paragraph 6
+    val text = """
+                 |“The levels of malnutrition among children continue to be truly alarming,” said Mahimbo Mdoe,
+                 |UNICEF’s Representative in South Sudan.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 6"
+
+    ignore should "have correct output #6" taggedAs(Somebody) in {
+      val malnutrition = NodeSpec("levels of malnutrition", Quant("truly", "alarming"))
+
+      tester.test(malnutrition) should be (successful)
+    }
+  }
+
+  { // Paragraph 7
+    val text = """
+                 |We have started ramping up food and nutrition support, but much more is needed to keep things
+                 |from deteriorating even further during the lean season,” said WFP Country Director Joyce Luma.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 7"
+
+    ignore should "have correct output #7" taggedAs(Somebody) in {
+      val support = NodeSpec("food and nutrition support", Inc("ramping"))
+
+      tester.test(support) should be (successful)
+    }
+  }
+
+  { // Paragraph 8
+    val text = """
+                 |“We are now seeing sharp spikes of need in new areas, such as Eastern Equatoria or Western
+                 |Bahr el-Ghazal, where malnutrition rates in some places are reaching dangerous levels.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 8"
+
+    ignore should "have correct output #8" taggedAs(Somebody) in {
+      val need = NodeSpec("need", Inc("sharp", "spikes"))
+      val malnutrition = NodeSpec("malnutrition rates") // TODO: what does "reaching dangerous levels" mean: Inc or Dec?
+
+      tester.test(need) should be (successful)
+      tester.test(malnutrition) should be (successful)
+    }
+  }
+
+  { // Paragraph 9
+    val text = """
+                 |The dramatic rise in malnutrition rates, means that in the first four months of the year UNICEF
+                 |has already treated 45 per cent of its planned 2016 caseload of 166,000 children.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 9"
+
+    ignore should "have correct output #9" taggedAs(Somebody) in {
+      val malnutrition = NodeSpec("malnutrition rates", Inc("rise"), Quant("dramatic"))
```

Comment:
I know I havent' been consistent and the system's not 100% but *ideally* here we'd have ` NodeSpec("malnutrition rates", Inc("rise", "dramatic"))`... is there a correlation event here too...? the thing being correlated with is lame and hard to get though...